setup.py

```diff
@@ -81,13 +81,12 @@ def load_deps(path):
         "boto3~=1.9, <1.17.107",
         "botocore>=1.20.106,<1.20.107",
         "aiobotocore~=1.4.0",
-        "aioitertools<0.9",
         "s3fs~=2021.8.1",
     ],
     "azure-blob-storage": [
         "msrest~=0.6.21",
-        "azure-storage-blob~=12.0",
-        "azure-core<1.23",
+        "azure-storage-blob>=12.13",
+        "azure-core>=1.23",
```

Comment:
I guess it should be
```suggestion
        "azure-storage-blob~=12.13",
        "azure-core~=1.23",
```